python/example_code/lookoutvision/datasets.py

```diff
@@ -0,0 +1,282 @@
+            "auto-label-metadata": {
+                "confidence": 1,
+                "job-name": "labeling-job/auto-label",
+                "class-name": class_name,
+                "human-annotated": "yes",
+                "creation-date": dttm,
+                "type": "groundtruth/image-classification",
+            },
+        }
+        return manifest
+
+    @staticmethod
+    def delete_dataset(lookoutvision_client, project_name, dataset_type):
+        """
+        Deletes an Amazon Lookout for Vision dataset
+        param: lookoutvision_client: The Amazon Lookout for Vision boto 3 client.
+        param: project_name: The name of the project that contains the dataset that
+        you want to delete.
+        param: dataset_type: The type (train or test) of the dataset that you
+        want to delete.
+        """
+        try:
+            lookoutvision_client = boto3.client("lookoutvision")
+
+            # Delete the dataset
+            logger.info(
+                "Deleting the " + dataset_type + " dataset for project " + project_name
+            )
+            lookoutvision_client.delete_dataset(
+                ProjectName=project_name, DatasetType=dataset_type
+            )
+            logger.info("Dataset deleted")
+
+        except ClientError as err:
+            logger.exception(
+                "Service error: Couldn't delete dataset: %s", err.response["Message"]
+            )
+            raise
+
+    @staticmethod
+    def describe_dataset(lookoutvision_client, project_name, dataset_type):
+        """
+        Gets information about an Amazon Lookout for Vision dataset.
+        param: lookoutvision_client: The Amazon Lookout for Vision boto3 client.
+        param: project_name: The name of the project that contains the dataset that
+        you want to describe.
+        param: dataset_type: The type (train or test) of the dataset that you want
+        to describe.
+        """
+
+        try:
+            # Describe a dataset
+
+            response = lookoutvision_client.describe_dataset(
+                ProjectName=project_name, DatasetType=dataset_type
+            )
+            print("Name: " + response["DatasetDescription"]["ProjectName"])
+            print("Type: " + response["DatasetDescription"]["DatasetType"])
+            print("Status: " + response["DatasetDescription"]["Status"])
+            print("Message: " + response["DatasetDescription"]["StatusMessage"])
+            print(
+                "Images: " + str(response["DatasetDescription"]["ImageStats"]["Total"])
+            )
+            print(
+                "Labeled: "
+                + str(response["DatasetDescription"]["ImageStats"]["Labeled"])
+            )
+            print(
+                "Normal: " + str(response["DatasetDescription"]["ImageStats"]["Normal"])
+            )
+            print(
+                "Anomaly: "
+                + str(response["DatasetDescription"]["ImageStats"]["Anomaly"])
+            )
```

Comment:
Prefer f-strings outside of logger.
Or, because you're pretty much just printing the whole structure, you can print the formatted object in one line with pprint:
```
from pprint import pprint
pprint(response["DatasetDescription"])
```

Reply:
Fixed.